jupyterhub/spawner.py

```diff
@@ -768,9 +783,9 @@ def get_env(self):
         # Info previously passed on args
         env['JUPYTERHUB_USER'] = self.user.name
         env['JUPYTERHUB_SERVER_NAME'] = self.name
-        env['JUPYTERHUB_API_URL'] = self.hub.api_url
+        env['JUPYTERHUB_API_URL'] = self.hub_connect_url
```

Comment:
This is not treated the same as JuptyerHub.hub_connect_url, and it probably should be based on the description. In which case, this should be `url_path_join(self.hub_connect_url, 'hub/api')` and ACTIVITY_URL should match.

Reply:
I'm a little bit confused. Do you think asking a user to set a URL with `/hub/api` or just URL w/o path on `hub_connect_url`?

Reply:
To match `hub_connect_url` it should not include the `hub/api` path